shared/chat/conversation/messages/react-button/emoji-picker/container.tsx

```diff
@@ -67,13 +69,13 @@ const useSkinTone = () => {
   )
   return {currentSkinTone, setSkinTone}
 }
-const useCustomReacji = (conversationIDKey: Types.ConversationIDKey) => {
+const useCustomReacji = (conversationIDKey: Types.ConversationIDKey, onlyInTeam: boolean | undefined) => {
```

Comment:
could also be `onlyInTeam?: boolean` since there's no required arg afterwards, but not a big deal